src/components/ErrorPage/ErrorPage.module.css

```diff
@@ -0,0 +1,3 @@
+.return-button-margin {
+  margin-top: 15px;
```

Comment:
for generic margins, paddings, etc, we use so called generics (see index.css), let's use `m-t-15: 15px` and remove this css module